### DISCUSSION TITLE
Monika "backing up" conversation

### DISCUSSION THREAD

**Discussion by Kovakhiin** [Monika-After-Story/MonikaModDev#9534] [QA - tech support + general] [answered]
Can you please make where the MAS persistent saves a bit more clearer when Monika mentions it, or maybe have it save a local one in the ddlc-win file? I'm only asking because I didn't realize they were saved in appdata and I backed up the wrong DDLC persistent, losing my Monika from 26-12-21-now, after I got a replacement PC.

**Comment by Booplicate** [comment, marked as answer]
In the conversation Monika shows a link to [the guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-back-up-my-persistent).
